docs/storage-management.asciidoc

```diff
@@ -3,67 +3,159 @@
 
 [partintro]
 --
-In the following section you can learn how to:
 
-* <<manage-indices-kibana, Manage APM indices via Kibana>>
+* <<sizing-guide, Storage and sizing guide>>
+* <<processing-performance, Processing and performance>>
 * <<reduce-storage, Reduce storage usage>>
+* <<manage-indices-kibana, Manage APM indices via Kibana>>
 * <<update-existing-data, Update existing data>>
 --
 
-[[manage-indices-kibana]]
-== Manage Indices via Kibana
-The Kibana UI for managing indices allows you to
-view indices, index settings, mappings, docs count, used storage per index and much more.
-You can perform management operations,
-for example, deleting indices directly via the Kibana UI.
-The UI also supports applying bulk operations on several indices at once.
-Check out the {kibana-ref}/managing-indices.html[Kibana Managing Indices] documentation for more details.
+[[sizing-guide]]
+== Storage and sizing guide
+
+APM Server handles a few different kinds of {apm-overview-ref-v}/apm-data-model.html[events].
+Processing and storage costs are largely dominated by
+{apm-overview-ref-v}/transactions.html[transactions] and
+{apm-overview-ref-v}/transaction-spans.html[spans],
+so only those will be covered here.
+
+[float]
+[[typical-transactions]]
+=== Typical transactions
+
+The size of a transaction depends on the language, agent settings, and what services the agent instruments.
+For instance, a service with a popular tech stack (web framework, database, caching library, etc.) is more likely to generate bigger transactions.
+
+In addition, all agents support manual instrumentation.
+How little or much you use these APIs will also impact what a typical transaction looks like.
+
+Due to the high variability of transactions, it's difficult to classify a transaction as typical.
+Regardless, here's a speculative reference: 
+
+[options="header"]
+|=======================================================================
+|Transaction size |Number of Spans |Number of Frames
+|_Small_ |5-10 |5-10
+|_Medium_ |15-20 |15-20
+|_Large_ |30-40 |30-40
+|=======================================================================
+
+There will always be transaction outliers with hundreds of spans or frames, but those are very rare.
+Most customers should have _small_ transactions.
```

Comment:
Most customers should have _small_ transactions => Small transactions are the most common.